.github/workflows/update-openapi.yml

```
@@ -0,0 +1,159 @@
+            # Generate a checksum of the content for PR comparison
+            CHECKSUM=$(cat tmp/openapi.json | sha256sum | cut -d' ' -f1)
+            echo "checksum=$CHECKSUM" >> $GITHUB_OUTPUT
+          fi
+      
+      - name: Check for existing PRs
+        if: steps.fetch-spec.outputs.has_changes == 'true'
+        id: check-prs
+        run: |
+          # Create a comment with the checksum to identify PRs with the same content
+          CHECKSUM_COMMENT="OpenAPI-Checksum: ${{ steps.fetch-spec.outputs.checksum }}"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)